2020/Pwn/pwncry/admin/README.md

```diff
@@ -0,0 +1,7 @@
+# InCTFi-pwncypto-chall
```

Comment:
@souragc inconsistency in the name of the challenge. The heading should be `# pwncry`.